### ISSUE TITLE
TILDE character ~ is replaced with SMALL TILDE ˜

### ISSUE THREAD

**Issue by daniel-afana** [saadq/latexresu.me#35]
The user puts a tilde ~ in any field in any section, then it always gets replaced with a small tilde ˜ when the user makes a preview. 
It's important, because, for instance, Upwork profiles urls contain tilda: https://www.upwork.com/freelancers/~profileID

![latex](https://user-images.githubusercontent.com/27088418/33460751-b1df9f06-d640-11e7-9b1c-375df430e3cb.png)
![latex-preview](https://user-images.githubusercontent.com/27088418/33460753-b4f61292-d640-11e7-98d5-1fe19d188542.png)

**Comment by saadq** [comment]
Yes, I am able to replicate this issue. When I sanitize the form inputs, I do a replace of certain problematic characters, including `~` as you can see [here](https://github.com/saadq/sanitize-latex/blob/master/index.js#L14). However, it seems like `\textasciitilde{}` can cause some issues with certain TeX fonts, so I will need to investigate a bit into this. Thanks for reporting this issue.
